draft-gazdag-x509-hash-sigs.md

```diff
@@ -144,7 +167,33 @@ i.e. either 24, 32 or 64 bytes. The height of a single tree is typically given
 by the parameter 'h'. The number of levels of trees is either called 'L' (HSS)
 or 'd' (XMSS, XMSS^MT, SPHINCS+).
 
-# Subject Public Key Algorithms
+# Use Cases of HBS in X.509
+
+As many cryptographic algorithms that are considered to be quantum-resistant,
+HBS have several pros and cons regarding their practical usage. On the positive side
+they are considered to be secure against a classical as well as a quantum adversary,
+a secure instantiation of HBS may always be built as long as a cryptographically
```

Comment:
", a secure instantiation of HBS" ersetzen durch ", and a secure instantiation of HBS"